composeApp/src/commonMain/kotlin/sp/bvantur/tasky/login/ui/LoginScreen.kt

```diff
@@ -32,8 +39,11 @@ fun LoginRoute() {
     LoginScreen()
 }
 
+@OptIn(ExperimentalComposeUiApi::class)
 @Composable
 fun LoginScreen() {
+    val (passwordRequester) = FocusRequester.createRefs()
```

Comment:
With each recomposition, a new instance will be created, which is not what you want. To maintain the same state across compositions, wrap it in the `remember{}` block.

Also, the `createRefs()` API is convenient when creating multiple `FocusRequester`s. Since you only need one, you can use the `FocusRequester()` constructor.

```
val passwordRequester = remember { FocusRequester() }
```

Reply:
Good catch. Will fix that ASAP. Thanks! :)